Loop/Managers/DeviceDataManager.swift

```diff
@@ -533,6 +560,59 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate {
         }
     }
 
+    // MARK: - Share Receiver
+
+    // MARK: ReceiverDelegate
+
+    var latestGlucoseG4: GlucoseG4?
+
+    func receiver(receiver: Receiver, didReadGlucoseHistory glucoseHistory: [GlucoseG4]) {
+        assertCurrentPumpData()
+
+        guard let latest = glucoseHistory.sort({ (a, b) in a.sequence < b.sequence }).last where latest != latestGlucoseG4 else {
```

Comment:
Tip: you can write `{ (a, b) in a.sequence < b.sequence }` as `{ $0.sequence < $1.sequence }`. You can also implement the `Comparable` protocol on `GlucoseG4` to use the `sequence` attribute, and then you can simply write `glucoseHistory.sort(<)`